docs/source/usrman/mpi4py.futures.rst

```diff
@@ -445,14 +466,9 @@ As explained previously, the 17 processes are partitioned in one master and 16
 workers. The master process executes the main script while the workers execute
 the tasks submitted from the master.
 
-.. [#] This :program:`mpiexec` invocation example using the ``-usize`` flag
-   (alternatively, setting the :envvar:`MPIEXEC_UNIVERSE_SIZE` environment
-   variable) assumes the backend MPI implementation is an MPICH derivative
-   using the Hydra process manager. In the Open MPI implementation, the MPI
-   universe size can be specified by setting the :envvar:`OMPI_UNIVERSE_SIZE`
-   environment variable to a positive integer.  Check the documentation of your
-   actual MPI implementation and/or batch system for the ways to specify the
-   desired MPI universe size.
+.. [#] When using MPI implementations different from MPICH and Open MPI, please
+   check the documentation of your actual MPI implementation and/or batch
```

Comment:
And here he have "MPI implementation" again... There is a reason I don't write docs, getting it right is hard.

Reply:
Do you mean the repetition of "MPI implementation" in the same sentence? I do not see at as a problem.